easybuild/easyconfigs/r/reframe/reframe-3.0-dev3.eb

```diff
@@ -0,0 +1,44 @@
+# @author: gppezzi, vkarak, manitart
+
+easyblock = 'Tarball'
+
+name = 'reframe'
+version = '3.0-dev3'
+
+homepage = 'https://github.com/eth-cscs/reframe'
+description = 'A regression framework for writing portable tests for HPC systems.'
+
+toolchain = SYSTEM
+
+sources = [
+    {
+        'filename': 'v%(version)s.tar.gz',
+        'source_urls': ['https://github.com/eth-cscs/reframe/archive/']
+    },
+    {
+        'filename': '/apps/common/easybuild/sources/%(nameletterlower)s/'
+                    '%(name)s/reframe-tests-%(version)s.tar.gz'
+    }
+]
+
+builddependencies = [('pip', '20.0.2', '-py3')]
+
+keepsymlinks = True
+postinstallcmds = [
+    'cp %(installdir)s/config/cscs.py %(installdir)s/reframe/settings.py',
+    'cp -r %(builddir)s/reframe-tests-%(version)s/checks %(installdir)s/cscs-checks/private',
+    'mv %(installdir)s/cscs-checks %(installdir)s/checks',
+    'PYTHONUSERBASE=%(installdir)s/installed_by_pip pip install --user -r %(builddir)s/reframe-%(version)s/requirements.txt',
+]
+
+modtclfooter = '''
+prepend-path PYTHONPATH %(installdir)s/installed_by_pip/lib/python3.6/site-packages
```

Comment:
I think it is more portable if you use 
```
modextrapaths = {
    'PYTHONPATH': '...'
}

Reply:
Yes, I had it in mind but did not include the change, thanks